### PR TITLE
Visual improvements

### PR DESCRIPTION
**Pull request by 7091lapS**
- Add more descriptive labels
- Add toggle to switch between absolute and percent data
- Add color coded title for better clarity
screen shot 2017-08-21 at 16 13 06
screen shot 2017-08-21 at 16 13 28

### CODE REVIEW

File: public/js/helpers/chartsHelpers.js [hunk 1]
```diff
@@ -21,8 +21,8 @@ const createYTotalsList = (partialsList) => {
 const convertToPercentage = (value, total) => total ? value * 100 / total : 0;
 
 // Converts the y value in a data pair to its percentage value (using the y totals value)
-const percentageDataSetPair = (pair, total) => {
-    return {'x': pair['x'], 'y': convertToPercentage(pair['y'], total)};
+const percentageDataSetPair = (pair, total) => { 
+    return { x: pair['x'], y: convertToPercentage(pair['y'], total), label: pair['label'] };
```
Comment (akash1810) [inline public/js/helpers/chartsHelpers.js]: minor: could use dot notation here, `pair.label` instead of `pair['label']` for example

File: public/js/services/Api.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ import moment from 'moment';
 export const httpClient = axios.create({
     // baseURL: 'https://productionmetrics.local.dev-gutools.co.uk/', // Use this url in dev. TODO: Add proper ENV management
     baseURL: 'https://productionmetrics.gutools.co.uk/',
-    timeout: 5000,
+    timeout: 10000,
```
Comment (akash1810) [inline public/js/services/Api.js]: why this change?

Comment (7091lapS) [inline public/js/services/Api.js]: The internet was very slow and i was getting timeout errors. I can probably revert this now :)

File: public/js/components/Charts/AreaChartWrap.js [hunk 3]
```diff
@@ -15,20 +16,28 @@ const AreaChartWrap = ({ title, series, xLabel, yLabel, isUpdating, scale, stack
     
     return (
         <div className={getClassName()}>
+            <h3>Tool of Origin: <span style={{color: '#33a22b'}}>InCopy</span> vs <span style={{color: '#ffbc01'}}>Composer</span></h3>
```
Comment (susiecoleman) [inline public/js/components/Charts/AreaChartWrap.js]: Does it make more sense for the style to be in classes in css? So we can use a value in `_variables.scss` and if the graph colours change the title changes as well

File: public/styles/components/_chart.scss [hunk 4]
```diff
@@ -3,6 +3,12 @@ $loadingLabelHeight: 300px;
 
 .chart-wrap {
   position: relative;
+  background: #393938;
+  border-top: 1px solid #ffbc01;
```
Comment (susiecoleman) [inline public/styles/components/_chart.scss]: This is one of the generic colours in the `_variables.scss` file. It would probably be better to refer to this if possible.
